### PR TITLE
adding sil_p_geez

### PR DESCRIPTION
**Pull request by rik-shaw**

### CODE REVIEW

**Comment by rik-shaw** [comment]
First attempt at adding a sil_p_geez keyboard.

File: sil_p_geez/sil_p_geez.kpj [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<KeymanDeveloperProject>
+  <Options>
+    <BuildPath></BuildPath>
```
Comment (mcdurdin) [inline sil_p_geez/sil_p_geez.kpj]: Set this to $PROJECTPATH\build

File: sil_p_geez/sil_p_geez.kpj [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<KeymanDeveloperProject>
+  <Options>
+    <BuildPath></BuildPath>
+    <CompilerWarningsAsErrors>False</CompilerWarningsAsErrors>
```
Comment (mcdurdin) [inline sil_p_geez/sil_p_geez.kpj]: Please set this to True unless your keyboard has unresolveable warnings.

File: sil_p_geez/sil_p_geez.kpj [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<KeymanDeveloperProject>
+  <Options>
+    <BuildPath></BuildPath>
+    <CompilerWarningsAsErrors>False</CompilerWarningsAsErrors>
+    <WarnDeprecatedCode>True</WarnDeprecatedCode>
+  </Options>
+  <Files>
+    <File>
```
Comment (mcdurdin) [inline sil_p_geez/sil_p_geez.kpj]: Please add the source\sil-p-geez.kmn file to the project

File: sil_p_geez/source/sil-p-geez.kmn [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+store(&COPYRIGHT) '© 2016 SIL Ethiopia'
+MESSAGE "The SIL P-Ge'ez Unicode keyboard is distributed under the GNU General Public License Version 3 (https://www.gnu.org/licenses/gpl-3.0.en.html)."
+
+c ***********************************************************************************
+c Amharic fidel
+c ***********************************************************************************
+
+c ----------------------------------------------------------------------------------
+c This new store (24-Sept-2007) is a fix for the problem of fast typing reordering keystrokes when using Keyman with TSF
+store(chars) 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+-=[]\{}|;:",./<>?' "'`~"
```
Comment (mcdurdin) [inline sil_p_geez/source/sil-p-geez.kmn]: This should no longer be required -- this issue was resolved a while back.

File: sil_p_geez/source/sil-p-geez.kps [hunk 5]
```diff
@@ -0,0 +1,2 @@
...
.../MSIFileName><MSIOptions></MSIOptions></Options><Buttons><Button><ButtonType>install</ButtonType><Caption>&amp;Install</Caption><URL></URL><Left>0</Left><Top>0</Top></Button><Button><ButtonType>cancel</ButtonType><Caption>Cancel</Caption><URL></URL><Left>0</Left><Top>0</Top></Button><Button><ButtonType>about</ButtonType><Caption>&amp;About...</Caption><URL></URL><Left>0</Left><Top>0</Top></Button></Buttons><Registry/><StartMenu><Folder></Folder><Items/></StartMenu><Info><Name URL="">SIL-P-Geez</Name><Version URL="">1.0</Version><Copyright URL="">GNU General Public License Version 3</Copyright><Author URL="mailto:wasta.linux.team@gmail.com">Wasta Linux Team</Author><WebSite URL="www.wastalinux.org">www.wastalinux.org</WebSite></Info><Files><File><Name>sil-p-geez.htm</Name><Description>File sil-p-geez.htm</Description><CopyLocation>0</CopyLocation><FileType>.htm</FileType></File><File><Name>sil-p-geez.kmn</Name><Description>File sil-p-geez.kmn</Description><CopyLocation>0</CopyLocation><FileType>.kmn</FileType></File><File><Name>sil-p-geez.png</Name><Description>File sil-p-geez.png</Description><CopyLocation>0</CopyLocation><FileType>.png</FileType></File></Files><Strings/></Package>
```
Comment (mcdurdin) [inline sil_p_geez/source/sil-p-geez.kps]: You won't need the .kmn referenced in the .kps file, but you will need the .kmx (from the build/ folder)

**Comment by rik-shaw** [comment]
@mcdurdin I have renamed to "sil_ethiopic_power_g" and hopefully have everything right now.  So the "official name" is "SIL Ethiopic Power G" to comply, but in my .htm file I refer to it as "SIL Ethiopic Power-G" so that "G" isn't all lonely.

Interestingly, it seems that Keyman Developer is wiping out the <BuildPath> and resetting the <CompilerWarningsAsErrors> back to "False".  I fixed them up manually.

Also, the reason my .kmn had a bad encoding from before was as you guessed: I had developed the .kmn in Linux (without a UTF-8 Preamble), but then when imported into Keyman Developer it got it mixed up and when I saved it there it changed the encoding.  But if I start an empty file with Keyman Developer it does save it as UTF-8 correctly.

**Comment by mcdurdin** [comment]
I made three changes when I accepted this:

1. Removed commas from the ethnologuecode system store contents
2. Fixed up the path of the .kmx in the .kps file to point to the build folder
3. Removed the 'desktop' build target as this one currently causes a web build which is not currently supported with this keyboard

**Pull request merged**